libmamba/src/api/configuration.cpp

```diff
@@ -1007,14 +1007,24 @@ namespace mamba
         {
             // Prepend all the directories in the environment variable `CONDA_ENVS_PATH`.
             auto conda_envs_path = util::get_env("CONDA_ENVS_PATH");
+
             if (conda_envs_path)
             {
+                if (util::get_env("CONDA_ENVS_DIRS"))
+                {
+                    LOG_ERROR << "The `CONDA_ENVS_DIRS` and `CONDA_ENVS_PATH` environment variables are both set, but only one must be declared.\nWe recommend setting `CONDA_ENVS_DIRS` only.)";
```

Comment:
```suggestion
                    LOG_ERROR << "The `CONDA_ENVS_DIRS` and `CONDA_ENVS_PATH` environment variables are both set, but only one must be declared.\nWe recommend setting `CONDA_ENVS_DIRS` only.";
```

Reply:
Whoops! Code is easy, English is hard!

Reply:
Probably due to one of my previous comment.